packages/react/src/isomorphicClerk.ts

```diff
@@ -947,6 +985,13 @@ export class IsomorphicClerk implements IsomorphicLoadedClerk {
     }
   };
 
+  authenticateWithGoogleOneTap = async (
+    params: AuthenticateWithGoogleOneTapParams,
+  ): Promise<SignInResource | SignUpResource> => {
+    const clerkjs = await this.#waitForClerkJS();
+    return clerkjs.authenticateWithGoogleOneTap(params);
+  };
```

Comment:
I want to highlight this change. This goes against the pattern we already have. But I cannot think of a reason why we cannot do this.